.github/workflows/sdk-features-integration.yml

```diff
@@ -0,0 +1,50 @@
+name: SDK Features Integration
+
+on: # rebuild any PRs and main branch changes
+  pull_request:
+  push:
+    branches:
+      - master
+
+concurrency: # Auto-cancel existing runs in the PR when a new commit is pushed
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
```

Comment:
I think need a fallback scenario for `main` builds: https://docs.github.com/en/actions/using-jobs/using-concurrency#example-using-a-fallback-value.

```
${{ github.head_ref || github.run_id }}
```

Reply:
Yeah, it does, thanks.